applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -92,37 +92,6 @@ static std::ostream& pythonToSofaDataString(PyObject* value, std::ostream& out)
         return out;
     }
 
-    if( PySequence_Check(value) )
```

Comment:
Ok, I was not aware of the whole discussion, but from discussion here it looks the right way to me. If a sofa type needs to implement its own python serialization, then it is a matter to implement the __repr__() function for that type. It sounds more standard, doesn't it ?